apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -44,11 +44,13 @@
   "filterHeaderDefault": "Sort",
   "filterHeaderDisplayWeight": "Top rated",
   "filterHeaderPopularityRank": "Most popular",
-  "roboticsButtonText": "Got robots? <a href='{robotics_url}'>Use these activities</a> and make a tangible Hour of Code for students of any age!",
+  "roboticsButtonText": "View robotics activities",
+  "roboticsText": "Got robots? <a href='{robotics_url}'>Use these activities</a> and make a tangible Hour of Code for students of any age!",
   "tutorialDetailsTeacherNotes": "Teacher notes",
   "tutorialDetailsShortLink": "Short link",
   "tutorialDetailInternationalLanguages": "Languages",
+  "tutorialDetailStandards": "Standards",
   "headingTutorialsYourLanguage": "Tutorials in your language",
   "headingTutorialsManyLanguages": "Tutorials in many languages",
-  "tutorialDetailStandards": "Standards"
+  "bottomGuidelinesLink": "To see our criteria and submission guidelines for Hour of Code tutorials, <a href='{guidelines_url}'>click here</a>."
```

Comment:
This string it seems like we could potentially split into two strings (one of which is just `click here` and then be able to have the anchor tag live in JSX instead of our string.